taskfile/taskfile.go

```diff
@@ -117,7 +115,6 @@ func Exists(l *logger.Logger, path string) (string, error) {
 	for _, taskfile := range defaultTaskfiles {
 		alt := filepathext.SmartJoin(path, taskfile)
 		if _, err := os.Stat(alt); err == nil {
-			l.VerboseOutf(logger.Magenta, "task: [%s] Not found - Using alternative (%s)\n", path, taskfile)
```

Comment:
I’m OK with removing it because, even though we could use it for troubleshooting, I don’t think it’s used at all. I just wanted to make sure it wasn’t an oversight.

Reply:
Yeah, its difficult to log here without passing a function in. If we really want the logs, we could compare the input/output in the caller and log there instead, but I decided it wasn't worth it.

Reply:
I agree!